samples/bluetooth/cap_acceptor/README.rst

```diff
@@ -0,0 +1,74 @@
+.. zephyr:code-sample:: bluetooth_cap_acceptor
+   :name: Bluetooth: Common Audio Profile Acceptor
+   :relevant-api: bt_cap bt_bap bluetooth
+
+   Bluetooth: Common Audio Profile Acceptor
```

Comment:
Just noticed this one: this should actually be a short description of what the sample is about. It is recommended to word this as if you were completing the sentence "This code sample shows how to ...". Otherwise https://builds.zephyrproject.io/zephyr/pr/72075/docs/connectivity/bluetooth/api/audio/cap.html is not as useful as it could be :)

![image](https://github.com/zephyrproject-rtos/zephyr/assets/128251/a34b4ed4-1e01-48c4-9525-0e0d06521b05)


So maybe: 

```suggestion
   Receive audio data from a Bluetooth device using the Common Audio Profile (CAP) Acceptor role
```

don't know if you maybe also want to have "unicast" as a keyword in that description as it looks like the sample might do multicast in the future?

Reply:
Yeah, it will support broadcast soon as well.